jaeger/jaeger_install/rhbjaeger-deploying.adoc

```diff
@@ -0,0 +1,65 @@
+[id="deploying-jaeger"]
+= Deploying Jaeger
+include::modules/ossm-document-attributes.adoc[]
+:context: jaeger-deploying
+
+toc::[]
+
+The Jaeger Operator includes a custom resource definition (CRD) file that determines the architecture and configuration settings to use when creating the Jaeger custom resources (CR). You can either install the default configuration or modify the custom resource file to better suit your business requirements.
+
+Each Jaeger instance is associated with a deployment strategy. You define the strategy in a custom resource file and when you create a Jaeger instance the Operator uses the deployment strategy and configuration options in the CR file to create the objects necessary for the deployment.
+
+.Jaeger custom resource file showing deployment strategy
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: simple-prod
+spec:
+  strategy: production <1>
+----
+
+
+<1> The Jaeger Operator currently supports the following deployment strategies:
+
+* *allInOne* (Default) strategy - This strategy is intended for development, testing, and demo purposes. The main backend components, Agent, Collector and Query service, are all packaged into a single executable which is configured (by default) to use in-memory storage.
++
+[NOTE]
+====
+In-memory storage is not persistent, which means that if the Jaeger instance shuts down, restarts, or is replaced, that your trace data will be lost.  For persistent storage, use the production Elasticsearch deployment strategy.
+====
++
+* *production* strategy - The production strategy is intended for production environments, where long term storage of trace data is important, as well as a more scalable and highly available architecture is required. Each of the backend components is therefore deployed separately.  The Agent can be injected as a sidecar on the instrumented application or as a daemonset. The Query and Collector services are configured with a supported storage type - currently Elasticsearch. Multiple instances of each of these components can be provisioned as required for performance and resilience purposes. For more information about configuring Elasticsearch with {product-title}, see  xref:../../logging/config/cluster-logging-elasticsearch.adoc[Configuring Elasticsearch].
+
+* *streaming* strategy - The streaming strategy is designed to augment the production strategy by providing a streaming capability that effectively sits between the Collector and the backend storage (Elasticsearch). This provides the benefit of reducing the pressure on the backend storage, under high load situations, and enables other trace post-processing capabilities to tap into the real time span data directly from the streaming platform (Kafka).
```

Comment:
minor point - instead of Kafka (or as well as) do we want to mention AMQ Streams (i.e. the product name)?

Reply:
Yes, I didn't realize it was productized.